docs/development.rst

```diff
@@ -333,9 +333,11 @@ Checklist:
   do a fresh start based on that.
 - run tox and/or binary builds on all supported platforms via vagrant,
   check for test failures
-- create a release on PyPi::
+- create sdist, sign it, upload release to (test) PyPi::
```

Comment:
i do wonder if at a later point a follow-up for creating a "manylinux" based wheel might be sensible (statically linking the dependencies)

Reply:
At a later point maybe, but not as long as that uses a ancient linux for building, which is below borg's minimum requirements.

Reply:
i wonder if manylinux2010 is sufficient

Reply:
hmm, i suspect centos6 may be too ancient